concepts/enums/introduction.md

```diff
@@ -0,0 +1,80 @@
+# Introduction
+
+## Enumerations
+
+So far we have mostly handled numbers and strings.
+To model the real world, we might want a limited number of options that a variable can take on.
+If you work for a skateboard manufacturer the deck material can be selected from maple, bamboo, or plastic.
+
+You could use integers to encode those values, but would have to use extra code to check if there is an invalid value coming from the system for the material.
+This is a case for `enumerations`.
+
+## Example usage
+
+The specific term for this kind of enumeration is `scoped enumeration`.
+The deck material can be written like this.
+Note the `enum class` keyword and the `;` at the end of the definition:
+
```

Comment:
"like this." is followed by a note instead of "this" example.  
Is there a better way to phrase/structure/order the two sentences and the example?